lerna.json

```diff
@@ -4,5 +4,5 @@
   ],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "2.2.1"
+  "version": "2.2.2-cra-sso-support.3"
```

Comment:
I'm not following why are we binding to a specific version

Reply:
it's fixed in main